internal/tasks/pulp-push-disk-images/tests/mocks.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+set -ex
+
+# mocks to be injected into task step scripts
+
+function select-oci-auth() {
+    echo Mock select-oci-auth called with: $*
+}
+
+function oras() {
+    echo Mock oras called with: $*
+
+    if [[ "$*" != "pull --registry-config"* ]]; then
+        echo Error: Unexpected call to oras
+        exit 1
+    fi
+
+    if [[ "$*" == *"nonexistent-disk-image"* ]]; then
+        echo Simulating failing oras pull call
+        exit 1
+    fi
+
+    echo a > disk.qcow2
```

Comment:
Why `echo a`? I mean it's ok. But why not just `echo > file` (save newline) or if you only care that the files get created `touch file`?

Reply:
switched to `touch`